blocks/editable/index.js

```diff
@@ -60,15 +58,17 @@ function isLinkBoundary( fragment ) {
 		fragment.childNodes[ 0 ].text[ 0 ] === '\uFEFF';
 }
 
-function getLinkProperties( element ) {
-	return { value: element.getAttribute( 'href' ) || '', target: element.getAttribute( 'target' ) || '', node: element };
-}
-
 function getFormatProperties( formatName, parents ) {
-	return formatName === 'link' ? getLinkProperties( find( parents, node => node.nodeName.toLowerCase() === 'a' ) ) : {};
+	switch ( formatName ) {
+		case 'link' :
```

Comment:
could we add scoping here:

```js
case 'link': {
     // code here
  }
```

Switch cases do not scope let/const variables by default, it could create confusion with other "cases", it's a good practice to scope the "case" whenever we declare a variable there.